packages/features/ee/impersonation/lib/ImpersonationProvider.ts

```diff
@@ -38,6 +38,7 @@ const auditAndReturnNextUser = async (
     role: impersonatedUser.role,
     impersonatedByUID,
     belongsToActiveTeam: hasTeam,
+    organizationId: impersonatedUser.organizationId,
```

Comment:
I also guessed we need to be able to know orgId for impersonation

Reply:
Interesting how we will handle impersonation into orgs 🤔